Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -280,12 +287,9 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
   [_pendingRequests removeAllObjects];
 
   // Stop updating if not not observing
-  if (!_observingLocation) {
+  if (!!_observingLocation) {
```

Comment:
I use that pattern in Javascript a lot, where types are implicit. But `_observingLocation` is already of type `BOOL`. Is it still necessary in ObjC?